git_machete/git_operations.py

```diff
@@ -161,7 +161,7 @@ def remotes(self) -> List[str]:
         return self.__remotes_cached
 
     def get_url_of_remote(self, remote: str) -> str:
-        return self.popen_git("remote", "get-url", "--", remote).strip()
+        return self.popen_git("config", "--get", f"remote.{remote}.url").strip()
```

Comment:
Because `get-url` is not supported in git 1.8

Reply:
Whoops if you could extract this change to a separate PR

Reply:
I'm just looking into git sources... seems that it's only been added in 2015 lol

Reply:
git v2.5.1 or so

Reply:
moved to separate PR